MAINTAINERS.md

```diff
@@ -0,0 +1,17 @@
+# The Chaos Mesh Maintainers
+
+This file lists who are the maintainers of the Chaos Mesh project.
+
+In short, maintainers are people who are in charge of the maintenance of the Chaos Mesh project. This includes everything from day-to-day tasks such as issue triage to completing epics. Maintainers should be the first point of contact for security issues or other critical issues that require privacy not offered by the issue tracker.
+
+The maintainers of Chaos Mesh, along with their email and focus area, are listed below:
+
+## Senior Maintainers
+
+Name | Email | Focus
+----|---|---
+[Siddon Tang](https://github.com/siddontang) | [tl@pingcap.com](mailto:tl@pingcap.com) | Project Lead
+[Qiang Zhou](https://github.com/zhouqiang-cl) | [zhouqiang@pingcap.com](mailto:zhouqiang@pingcap.com) | Project Lead
+[CWen](https://github.com/cwen0) | [cwen@pingcap.com](mailto:cwen@pingcap.com) | Operator, Dashboard
+
+> *Please do not contact the maintainers listed above for any reason not directly related to the Chaos Mesh project. For issues, questions, or support, you are asked to **use the issue tracker whenever possible.***
```

Comment:
```suggestion
```

Reply:
Suggest removing this because we are in a very early stage and does not need to add restriction to contact.

Reply:
Suggest removing this because we are in a very early stage and does not need to add restriction to contact.